packages/web/src/common/hooks/useFeatureFlag.ts

```diff
@@ -20,17 +24,28 @@ import { StateWithRemoteConfig } from 'common/store/remote-config/slice'
 export const createUseFeatureFlagHook = <State extends StateWithRemoteConfig>(
   remoteConfigInstance: RemoteConfigInstance
 ) => (flag: FeatureFlags) => {
+  const overrideKey = `${FEATURE_FLAG_OVERRIDE_KEY}:${flag}`
   const configLoaded = useSelector((state: State) =>
     isRemoteConfigLoaded<State>(state)
   )
   const userIdFlag = flagCohortType[flag] === FeatureFlagCohortType.USER_ID
   const hasAccount = useSelector(getAccountUser)
   const shouldRecompute = userIdFlag ? hasAccount : true
+  const setOverride = (val: OverrideSetting) => {
+    if (val === null) return localStorage.removeItem(overrideKey)
```

Comment:
We don't want to depend directly on localstorage here in common right? Can we use the state context stuff? Or else this really shouldn't live in common.

Reply:
Oh good call. Yeah the `remoteConfigInstance` passed into here already has `getFeatureFlagSessionId` and `setFeatureFlagSessionId`. We could change those to simply be `getLocalStorageItem` and `setLocalStorageItem` and then expose/use those in this hook. @Kyle-Shanks lmk if you wanna sync on this so I can explain better